### PR TITLE
Feature/627 swap query selectors for refs

### PR DESCRIPTION
**Pull request by tomlovesgithub**

### CODE REVIEW

**Comment by tomlovesgithub** [comment]
> @tomlovesgithub you are working on a task that is CLOSED. (!?)
> If it is additional work, you should create a new one, and specify the work that you are carrying out.

This task shouldn't be closed, its a ticket i created as the form component referencing the DOM means that tests for ~8 component files which extend the form cant be started

> These are breaking changes.
> Did you test every form and every file that you changed or was this a Find & replace task?

It is a wide brush approach, so yes a find and replace job, but I had no shortage of ideas on how to approach this task, but i was limited in not knowing exactly what the querySelector functionality was doing/where it was needed/not needed.
How would you suggest approaching this issue?

File: src/views/Apply/FinalCheck/Review.vue [hunk 1]
```diff
@@ -1771,7 +1776,7 @@ export default {
     downloadAsPdf() {
       const pdf = new jsPDF();
       pdf.fromHTML(
-        document.querySelector('#download-as-pdf-div'),
+        this.$refs['download-as-pdf-div'],
```
Comment (YaaL) [inline src/views/Apply/FinalCheck/Review.vue:1779]: Good catch!

Comment (tomlovesgithub) [reply src/views/Apply/FinalCheck/Review.vue:1779]: 🕵️‍♂️

**Comment by YaaL** [comment]
@lloback it looks fine now.

**Pull request merged**